R/geom-bar.r

```diff
@@ -1,12 +1,12 @@
 #' Bar charts
 #'
-#' There are two types of bar charts: `geom_bar` makes the height of the
+#' There are two types of bar charts: `geom_bar()` makes the height of the
```

Comment:
Stylistic, but since we open with two types...: I would follow the colon with "`geom_bar()` and `geom_col()`. `geom_bar()` makes..."